.github/workflows/redirects_page.yml

```diff
@@ -0,0 +1,39 @@
+name: Generate and Deploy Redirects
+
+on:
+  workflow_dispatch:
+  schedule:
```

Comment:
`Package.toml`, presumably?

Reply:
The files that contain the URL, whatever they're called.

Reply:
Seems reasonable to me. It might still be good to have a once-daily cron job (because the jobs might fail sometimes, due to the usual GitHub flakiness). But yes, building on pushes to master that modify a `Package.toml` file seems like it would be a good approach.

Reply:
I've replaced the cron job with a path-filtered trigger on push to master. I don't think an additional cron job is necessary, the action is unlikely to fail and the worst that can happen is that an old repo link stays active a little longer or a new package isn't linked for a short time.